### PR TITLE
Add ability to dynamically load a DOM element from server.

### PR DESCRIPTION
**Pull request by jamohile**
**Demo**
https://jamohile-step-2020.uc.r.appspot.com/projects.html
If you go to a specific project (Projects > click one), you will see a message pop up in the comments section.

**Background**
As part of week-3's task of building a comment system, step 2 is to load a hardcoded message from the server.

**Work Done**
At this point, the website loads a static "Hello Jay!" message from the server and injects it into the DOM. This particular message is only a proof of concept and will be removed in the next step(s).

**Design Decisions**
This message was injected into an unrelated area of the markdown for now. This was done to reuse markup for the coming comment feature with minimum impact to the markup.

### CODE REVIEW

File: portfolio/src/main/webapp/js/project-detail.js [hunk 1]
```diff
@@ -26,4 +26,19 @@ for(let detail of project.detail){
...
+async function loadServerMessage(){
+    /** For now we espect to get response <h1>Hello Jay!</h1>*/
+    const response = await fetch("/data");
+    const message = await response.text();
+
+    /** This is just a temporary place to put this message. */
+    document.querySelector("#comments-list").innerHTML = message;
+}
+
+loadServerMessage();
```
Comment (mctavish) [inline portfolio/src/main/webapp/js/project-detail.js:44]: missing newline :P

File: portfolio/src/main/webapp/js/project-detail.js [hunk 2]
```diff
@@ -26,4 +26,19 @@ for(let detail of project.detail){
...
-}
\ No newline at end of file
+}
+
+/**
+ * A simple example of loading data from the server.
+ * This is a proof-of-concept, and will be replaced by a more complicated and useful method.
+ */
+async function loadServerMessage(){
+    /** For now we espect to get response <h1>Hello Jay!</h1>*/
```
Comment (mctavish) [inline portfolio/src/main/webapp/js/project-detail.js:36]: nit: expect

File: portfolio/src/main/webapp/js/project-detail.js (hunk 1 above)
Comment (jamohile) [reply portfolio/src/main/webapp/js/project-detail.js]: Done.

File: portfolio/src/main/webapp/js/project-detail.js (hunk 2 above)
Comment (jamohile) [reply portfolio/src/main/webapp/js/project-detail.js]: Fixed.

**Pull request merged**